filesystems/oss/src/main/scala/cromwell/filesystems/oss/batch/OssBatchIoCommand.scala

```diff
@@ -75,20 +75,23 @@ sealed trait OssBatchHeadCommand[T] extends OssBatchIoCommand[T, ObjectMetadata]
 
 case class OssBatchSizeCommand(override val file: OssPath) extends IoSizeCommand(file) with OssBatchHeadCommand[Long] {
   override def mapOssResponse(response: ObjectMetadata): Long = response.getContentLength
+  override def commandDescription: String = s"OssBatchSizeCommand.init file '$file'"
 }
 
 case class OssBatchEtagCommand(override val file: OssPath) extends IoHashCommand(file) with OssBatchHeadCommand[String] {
   override def mapOssResponse(response: ObjectMetadata): String = response.getETag
+  override def commandDescription: String = s"OssBatchEtagCommand.init file '$file'"
 }
 
 case class OssBatchTouchCommand(override val file: OssPath) extends IoTouchCommand(file) with OssBatchHeadCommand[Unit] {
   override def mapOssResponse(response: ObjectMetadata): Unit = ()
+  override def commandDescription: String = s"OssBatchTouchCommand.init file '$file'"
```

Comment:
TOL (spotted here but true throughout) - why are we appending `.init` to the command descriptions for all these various command types?

Reply:
Well to be honest it was in the logging branch so I just copied it over here. But now that you have pointed it out I had a 'oops' moment since `.init` no longer makes sense as we don't log these messages anymore upon initialization. So I will remove it.